src/dbscan.jl

```diff
@@ -15,6 +17,13 @@ type DbscanResult <: ClusteringResult
 end
 
 
+immutable Cluster
+  size::Int64

```

Comment:
Please use the standard 4-space indents for consistency with the rest of the code